core/src/main/scala/org/genivi/sota/core/Boot.scala

```diff
@@ -55,13 +58,14 @@ trait RviBoot {
   }
 
   def rviRoutes(db: Database, notifier: UpdateNotifier, namespaceDirective: Directive1[AuthedNamespaceScope]): Route = {
-      new WebService(notifier, resolverClient,
-        deviceRegistryClient, db, namespaceDirective, messageBusPublisher).route ~
-      startSotaServices(db)
+    val updateService = new UpdateService(notifier, deviceRegistryClient)
+    new WebService(updateService, resolverClient,
+      deviceRegistryClient, db, namespaceDirective, messageBusPublisher).route ~
+    startSotaServices(db)
```

Comment:
as we're stripped away the dependency on the user-profile, I also introduced making use of TUF in rvi mode – it's off by default though